### PR TITLE
Car filters

### PR DESCRIPTION
**Pull request by synthetiquely**
Implement car filters form and sorting form.

### CODE REVIEW

**Review by ElizavetaBuraya — APPROVED** [review]

File: src/redux/sagas/carFiltersSaga.ts [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  }
+}
+
+function* callFetchFilterResults(action: any) {
+  yield put({ type: actionTypes.SET_LOADING, payload: true });
+  yield put({ type: actionTypes.SET_SEARCH_ERROR, payload: '' });
+  try {
+    const sortingParams = select(getSortingParams);
+    const data = transformDataForSearch(action.payload, sortingParams);
+    // const response = yield call(api.filters.fetchResults, data);
```
Comment (AntonNarkevich) [inline src/redux/sagas/carFiltersSaga.ts:109]: Do we need this comments?

File: src/redux/sagas/carFiltersSaga.ts [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+function* callUpdateFilterResults() {
+  yield put({ type: actionTypes.SET_LOADING, payload: true });
+  yield put({ type: actionTypes.SET_SEARCH_ERROR, payload: '' });
+  try {
+    const filterParams = select(getFilterParams);
+    const sortingParams = select(getSortingParams);
+    const data = transformDataForSearch(filterParams, sortingParams);
+    // const response = yield call(api.filters.fetchResults, data);
+    // yield put({ type: HERE GOES ACTION, payload: HERE GOES PAYLOAD });
```
Comment (AntonNarkevich) [inline src/redux/sagas/carFiltersSaga.ts:126]: Same.

File: src/utils/carFiltersValidation.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  }
+};
+
+export const validateForm = (values: FormData): any => {
+  const errors: any = {};
+  if (values.markId.length === 0) {
+    errors.markId = 'Field is required';
+  }
+  if (values.yearFrom && !validateYears(values.yearFrom)) {
+    errors.yearFrom = 'Please fill in a valid year';
```
Comment (AntonNarkevich) [inline src/utils/carFiltersValidation.ts:47]: Aren't we using the translations library?

**Review by AntonNarkevich — CHANGES_REQUESTED** [review]
The PR looks good for me except for 2 minor things. Please take a look. And let's merge.

Also, we're going to involve someone to audit the whole FE part soon.

**Pull request merged**